setup.cfg

```diff
@@ -1,11 +1,107 @@
+[metadata]
+name = odl
+version = attr: odl.__version__
+description = Operator Discretization Library
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Jonas Adler, Holger Kohr, and the ODL Contributors
+author_email = odl@math.kth.se
+license = MPL-2.0
+license_file = LICENSE
+url = https://github.com/odlgroup/odl
+download_url = https://github.com/odlgroup/odl/archive/master.zip
+platforms = any
+keywords = research, development, mathematics, prototyping, imaging, tomography
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Mathematics
```

Comment:
Also `Topic :: Scientific/Engineering :: Medical Science Apps.`?

Reply:
Yeah why not.